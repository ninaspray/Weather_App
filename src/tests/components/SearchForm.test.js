import React from "react";
import { render, screen } from "@testing-library/react";
import SearchForm from "../../components/SearchForm";

describe("SearchForm", () => {
        const validProps = {
            searchText: "test",
            setSearchText:() => {},
            onSubmit: () => {},
        };
        it ("renders correctly", () => {
            const {asFragment} = render(
                <SearchForm
                searchText={validProps.searchText} setSearchText={validProps.setSearchText} onSubmit={validProps.onSubmit} />
            );
            expect(asFragment()).toMatchSnapshot();
        })});