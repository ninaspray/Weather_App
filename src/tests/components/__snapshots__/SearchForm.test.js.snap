// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Form renders correctly 1`] = `
<DocumentFragment>
  <div
    class="search-form"
  >
    <input
      type="text"
      value=""
    />
    <button
      type="submit"
    >
      Search
    </button>
  </div>
</DocumentFragment>
`;

exports[`SearchForm renders correctly 1`] = `
<DocumentFragment>
  <div
    class="search-form"
  >
    <input
      type="text"
      value="test"
    />
    <button
      type="submit"
    >
      Search
    </button>
  </div>
</DocumentFragment>
`;
